velox/exec/PartitionedOutput.h

```diff
@@ -47,17 +49,17 @@ class Destination {
     rows_.push_back(rows);
   }
 
-  BlockingReason advance(
+  virtual BlockingReason advance(
```

Comment:
PartitionedOutputBufferManager seems to be an implementation detail of the StreamingDestination. Should we move it there?